app/models/custom_value/bool_strategy.rb

```diff
@@ -0,0 +1,39 @@
+#-- encoding: UTF-8
+#-- copyright
+# OpenProject is a project management system.
+# Copyright (C) 2012-2015 the OpenProject Foundation (OPF)
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See doc/COPYRIGHT.rdoc for more details.
+#++
+
+class CustomValue::BoolStrategy < CustomValue::FormatStrategy
+  def typed_value
+    unless value.nil? || value.blank?
+      value == '1'
+    end
+  end
+
+  def validate_type_of_value
+  end
```

Comment:
Indeed!

`Pustekuchen` corresponds to `false`... Same for `Unicorn`.

I was not sure how safe it is to introduce such a validation, when there was no validation in the old code (breaking existing stuff).

But strictly speaking you are right and I should probably also restrict it.